docs/install.rst

```diff
@@ -51,7 +45,7 @@ Astropy also depends on other packages for optional features:
 - `bleach <https://bleach.readthedocs.io/>`_: Used to sanitize text when
   disabling HTML escaping in the :class:`~astropy.table.Table` HTML writer.
 
-- `mock <https://github.com/testing-cabal/mock>`_ (python <= 3.2) or `unittest.mock <https://docs.python.org/dev/library/unittest.mock.html>`_ (python > 3.3):
+- `unittest.mock <https://docs.python.org/dev/library/unittest.mock.html>`_ (python > 3.3):
```

Comment:
How can a built-in be an optional dependency?

Reply:
Yep, should be removed :)

Reply:
You can opt not to use it? 😉 (No, you're right.)